curator-framework/src/main/java/org/apache/curator/framework/api/BackgroundPathableQuietly.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.curator.framework.api;
+
+public interface BackgroundPathableQuietly<T> extends BackgroundPathable<T>, Quietly<BackgroundPathable<T>>
```

Comment:
Naming convention nit: should be "BackgroundPathQuietlyable" - Yes, I'm probably going insane.